contrib/format-excel/src/test/java/org/apache/drill/exec/store/excel/TestExcelFormat.java

```diff
@@ -382,6 +382,26 @@ public void testExplicitSomeQueryWithCompressedFile() throws Exception {
     new RowSetComparison(expected).verifyAndClearAll(results);
   }
 
+  @Test
+  public void testFileWithDoubleDates() throws Exception {
+    String sql = "SELECT `Close Date`, `Type` FROM dfs.`excel/comps.xlsx` WHERE style='Contemporary'";
+
+    RowSet results = client.queryBuilder().sql(sql).rowSet();
+    results.print();
```

Comment:
Please remove. Printing pollutes the output.

Reply:
Removed.